app/javascript/components/enrollment/steps/ExposureInformation.js

```diff
@@ -632,179 +468,64 @@ class Exposure extends React.Component {
-                              name="assigned_user"
-                              label="Apply this change to the entire household that this monitoree is responsible for"
-                              onChange={this.handlePropagatedFieldChange}
-                              checked={this.state.current.propagatedFields.assigned_user}
-                            />
-                          </Form.Group>
-                        )}
-                    </Form.Group>
-                    <Form.Group as={Col} md="8" controlId="exposure_risk_assessment" className="mb-2 pt-2">
-                      <Form.Label className="input-label">RISK ASSESSMENT{schema?.fields?.exposure_risk_assessment?._exclusive?.required && ' *'}</Form.Label>
-                      <Form.Control
-                        isInvalid={this.state.errors['exposure_risk_assessment']}
-                        as="select"
-                        size="lg"
-                        className="form-square"
-                        onChange={this.handleChange}
-                        value={this.state.current.patient.exposure_risk_assessment || ''}>
-                        <option></option>
-                        <option>High</option>
-                        <option>Medium</option>
-                        <option>Low</option>
-                        <option>No Identified Risk</option>
-                      </Form.Control>
-                      <Form.Control.Feedback className="d-block" type="invalid">
-                        {this.state.errors['exposure_risk_assessment']}
-                      </Form.Control.Feedback>
-                    </Form.Group>
-                    <Form.Group as={Col} md="16" controlId="monitoring_plan" className="mb-2 pt-2">
-                      <Form.Label className="input-label">MONITORING PLAN{schema?.fields?.monitoring_plan?._exclusive?.required && ' *'}</Form.Label>
-                      <Form.Control
-                        isInvalid={this.state.errors['monitoring_plan']}
-                        as="select"
-                        size="lg"
-                        className="form-square"
-                        onChange={this.handleChange}
-                        value={this.state.current.patient.monitoring_plan || ''}>
-                        <option>None</option>
-                        <option>Daily active monitoring</option>
-                        <option>Self-monitoring with public health supervision</option>
-                        <option>Self-monitoring with delegated supervision</option>
-                        <option>Self-observation</option>
-                      </Form.Control>
-                      <Form.Control.Feedback className="d-block" type="invalid">
-                        {this.state.errors['monitoring_plan']}
-                      </Form.Control.Feedback>
-                    </Form.Group>
-                  </Form.Row>
+                  {this.exposureFields()}
```

Comment:
not sure this needed to be broken out but it is cleaner.  can we call it `renderExposureFields` or `renderInputFields`?